src/api/app/controllers/webui/staging/workflows_controller.rb

```diff
@@ -47,7 +47,7 @@ def edit
   end
 
   def destroy
-    @staging_workflow = StagingWorkflow.find_by(id: params[:id])
+    @staging_workflow = ::Staging::Workflow.find_by(id: params[:id])
```

Comment:
This can be just `Workflow`? or it doesn't work?

Reply:
Nope, because it will look in webui then.

Reply:
Just double checked:
```
uninitialized constant Webui::Staging::WorkflowsController::Workflow
```